modules/local/clipkit.nf

```diff
@@ -25,8 +25,10 @@ process CLIPKIT {
     script:
     def args = task.ext.args ?: ''
     """
+    prefix=\$(basename "${fasta}" -mafft.fa)
```

Comment:
I see, so by just stripping the rest of the filename from the MSA you're accomplishing the same thing as if you used the orthogroup name from the meta map (originally provided in a tuple). will be a lot cleaner.